control-plane/csi-driver/src/bin/node/nodeplugin_nvme.rs

```diff
@@ -1,10 +1,68 @@
-use crate::dev::Device;
+use crate::dev::{Device, DeviceError};
 use grpc::csi_node_nvme::{
     nvme_operations_server::NvmeOperations, NvmeConnectRequest, NvmeConnectResponse,
 };
-use std::collections::HashMap;
+use once_cell::sync::OnceCell;
+use std::{
+    collections::{HashMap, HashSet},
+    sync::Mutex,
+};
 use tonic::Response;
-use tracing::{info, warn};
+use tracing::{error, info, trace, warn};
+
+static REQUEST_INVENTORY: OnceCell<Mutex<HashSet<String>>> = OnceCell::new();
```

Comment:
Can we make this more generic, into maybe a VOLUME_SERIALIZER ? And put it into its own module.
We can extract the VolumeId from the NQN which would allow us to reuse this throughout the csi-driver later.

Reply:
Okay. That sounds good.

Reply:
Done.